runtime/jcl/common/mgmtos.c

```diff
@@ -245,8 +245,14 @@ Java_com_ibm_lang_management_internal_SysinfoCpuTime_getCpuUtilizationImpl(JNIEn
 			break;
 		}
 	}
+#if defined(J9ZOS390)
+	result = (*env)->NewObject(env, clazz, tempMethod,
+			cpuTime.timestamp, cpuTime.cpuTime , cpuTime.numberOfCpus, myStatus, cpuTime.cpuLoad);
```

Comment:
What is the default value for `cpuTime.cpuLoad` from the OMR code?  Is it `-1` when not supported?  Is the `#ifdef` really required?

Reply:
Made the change to support cpuLoad for all platforms. Value is initialized to -1 in OMR and updated with real cpuLoad only for zos.  #ifdefs removed. 
Change applied in ffdf65f8be493319c906b24c60b752782364cb35 and the corresponding change in OMR.